guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/GuvnorM2Repository.java

```diff
@@ -435,8 +434,9 @@ private void deployPom(final GAV gav,
                 .collect(Collectors.toList());
     }
 
-    public static String getPomText(final String path) {
-        final File file = new File(M2_REPO_DIR,
+    public String getPomText(final String path) {
```

Comment:
You'll have to excuse me; I am probably a bit thick when it comes to understanding these changes. A User can be merrily editing a project in _their_ workspace. They've uploaded a new JAR to the Workbench's "Artifact Repository" and added it as a dependency to their project when they click "View pom" in the Project Editor's "Dependency grid" they'll get the pom from the "global" repository. This suggests the Workbench's "Artifact Repository" is considered "global". Is this correct?